src/Microsoft.Identity.Web.TokenAcquisition/PublicAPI/net9.0/PublicAPI.Unshipped.txt

```diff
@@ -1,5 +1,8 @@
 #nullable enable
+const Microsoft.Identity.Web.ClaimConstants.Password = "xms_password" -> string!
```

Comment:
What are these changes?

Reply:
#3140 introduced these but it wasn't built with net 9, so the associated warnings didn't show up